modules/payloads/singles/windows/x64/meterpreter_bind_named_pipe.rb

```diff
@@ -6,7 +6,7 @@
 
 module MetasploitModule
 
-  CachedSize = 200262
+  CachedSize = 200774
```

Comment:
Looks like this bytes size has gone up which I wouldn't have expected 🤔 

A quick bindiff shows we're compiling the init logging and log function:
![image](https://user-images.githubusercontent.com/60357436/166654943-43583d00-83c9-4644-96d9-4c21946b247d.png)

![image](https://user-images.githubusercontent.com/60357436/166654974-71169375-62b4-40d3-9a62-fb43b8983bee.png)

![image](https://user-images.githubusercontent.com/60357436/166654999-4ade9241-f062-4563-8fe8-ba3315482be9.png)

Since metasploit-payloads already has extra commits, we can land this and fix it separately